src/styles/blocks/page.css

```diff
@@ -0,0 +1,9 @@
+.page {
+  margin: 0;
+  font-family: Avenir, serif;
+  font-weight: 300;
+}
+
+.page__search {
+  margin-top: 20px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both) to avoid potential margin collapse. Since you have 'margin-top' here, ensure that any corresponding bottom margin is not set within related elements to maintain consistency.